test/test_grudge.py

```diff
@@ -33,7 +33,6 @@
 
 from arraycontext.container.traversal import thaw
 
-from meshmode import _acf           # noqa: F401
```

Comment:
For a bit of context, this was just here for debugging, e.g. if you want to run a single test with
```
python test_grudge.py 'test_something(_acf, param, param, param)'
```
instead of going through `pytest` somehow.

Reply:
Ah I've wondered what that did... I've gone ahead and put it back: https://github.com/inducer/grudge/pull/210/commits/286614f7c220f9a8524553526eb97094e59e005b